extensions-contrib/opentelemetry-emitter/README.md

```diff
@@ -0,0 +1,170 @@
+# OpenTracing Emitter
+
+The OpenTelemetry emitter generates OpenTelemetry Spans for queries.
+
+
+## Configuration
+
+### Enabling
+
+Load the plugin and enable the emitter in `common.runtime.properties`:
+
+Load the plugin:
+
+```
+druid.extensions.loadList=[..., "opentelemetry-emitter"]
+```
+
+Enable the emitter:
+
+```
+druid.emitter=opentelemetry
+```
+
+*_Don't forget to remove the line_ `druid.emitter=noop` _to avoid rewriting_.
+
+## Testing
+###Zipkin
+Run zipkin docker container
+```
+docker run -d -p 9411:9411 openzipkin/zipkin
+```
+Zipkin will be available at `http://localhost:9411`.
+
+Add zipkin dependency to `extensions-contrib/opentelemetry-emitter/pom.xml`:
+```
+<dependency>
+  <groupId>io.opentelemetry</groupId>
+  <artifactId>opentelemetry-exporter-zipkin</artifactId>
+  <version>${your.version}</version>
+</dependency>
+```
+
+Run:
+
+```
+mvn package
+tar -C /tmp -xf distribution/target/apache-druid-0.21.0-bin.tar.gz
+cd /tmp/apache-druid-0.21.0
+```
+
+
+Edit `conf/druid/single-server/micro-quickstart/_common/common.runtime.properties` to enable
+the emitter (see `Configuration` section above). Set the following properties:
+```
+otel.traces.exporter=zipkin
```

Comment:
It doesn't work if add it to `common.runtime.properties`
I set this property directly from the code:
`System.setProperty("otel.traces.exporter", "zipkin")`
@xvrl  Could you tell me please if you know how to do it better?

Reply:
Sure
Usually all settings are stored in config class using jackson annotations, so it's seems good option for our case